### PR TITLE
Ga/qa 3

### PR DESCRIPTION
**Pull request by g-a-v-i-n**
- Polishes Leap a bit more
- Misc spacing fixes
- Cleaned up the user invite tag component
- Added short explanatory copy about what makes a group private vs public

### CODE REVIEW

File: pkg/interface/src/views/apps/launch/app.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const ScrollbarLessBox = styled(Box)`
 
 export default function LaunchApp(props) {
   const [hashText, setHashText] = useState(props.baseHash);
-
+  console.log('it works')
```
Comment (matildepark) [inline pkg/interface/src/views/apps/launch/app.js:28]: loose console.log

**Review by matildepark — CHANGES_REQUESTED** [review]
Just needs the console.log string removed and the commits should read, eg. `leap: resolve spacing` etc. Cross-interface changes use `landscape:`

File: pkg/interface/src/views/apps/launch/app.js (hunk 1 above)
Comment (g-a-v-i-n) [reply pkg/interface/src/views/apps/launch/app.js]: sure thing, wasn't aware of the commit message convention

**Comment by matildepark** [comment]
Can you reword the previous commits as well?

**Comment by g-a-v-i-n** [comment]
Sounds complex and I have no idea how to do that

**Comment by matildepark** [comment]
s'not bad

`git merge-base release/next-userspace ga/qa-3` (if you branched off next-userspace; if off master, then master)

then `git rebase -i <that commit id>`

You'll see a bunch of 'pick's for the commits you made; change it to reword, save and quit (`:wq`) and then one at a time you can reword the commit, save and quit, then you just `git push -f ga/qa-3` the branch.

**Comment by g-a-v-i-n** [comment]
did that work?

**Comment by matildepark** [comment]
Looks like yes

**Review by matildepark — APPROVED** [review]

**Comment by g-a-v-i-n** [comment]
Sweet, thx for the tip

**Pull request merged**